volue_insight_timeseries/events.py

```diff
@@ -107,7 +106,7 @@ def __init__(self, sse_event):
         self._raw_event = sse_event
         try:
             self.json_data = json.loads(sse_event.data)
-        except:
+        except json.JSONDecodeError:
```

Comment:
Is `JSONDecodeError` the only exception that can be raised here? Agree the original version is too broad and not good practice, but we should make sure if there is not other exceptions that will be missing with this.

Reply:
You are right, this method can also throw the TypeError in case the `data` is neither str nor bytes. I will add this as well.